tests/Integration/Behaviour/Features/Context/Domain/CountryFeatureContext.php

```diff
@@ -27,11 +27,28 @@
 
 namespace Tests\Integration\Behaviour\Features\Context\Domain;
 
+use Behat\Gherkin\Node\TableNode;
+use Country;
+use PrestaShop\PrestaShop\Core\Domain\Country\Command\AddCountryCommand;
+use PrestaShop\PrestaShop\Core\Domain\Country\Exception\CountryException;
 use PrestaShop\PrestaShop\Core\Domain\Country\Exception\CountryNotFoundException;
 use RuntimeException;
+use Tests\Integration\Behaviour\Features\Context\CommonFeatureContext;
+use Tests\Integration\Behaviour\Features\Context\SharedStorage;
 
 class CountryFeatureContext extends AbstractDomainFeatureContext
 {
+    /**
+     * @var int default shop id from configs
+     */
+    private $defaultShopId;
+
+    public function __construct()
+    {
+        $configuration = CommonFeatureContext::getContainer()->get('prestashop.adapter.legacy.configuration');
+        $this->defaultShopId = $configuration->get('PS_SHOP_DEFAULT');
+    }
+
```

Comment:
We already have an `AbstractProductFeatureContext::getDefaultShopId()` method for this, but it should be moved into the `AbstractDomainFeatureContext` so that other contexts can use it

Can you please move this method, so that you can then use it in your context?